### PR TITLE
Add possibility to run single service in different namespace.

### PR DESCRIPTION
**Pull request by DTadrzak**
WIP
Signed-off-by: DTadrzak <daniel.tadrzak@intel.com>

### CODE REVIEW

**Comment by DTadrzak** [comment]
@PiotrProkop 
@pawelsocha

File: stackanetes/configmap.py [hunk 1]
```diff
@@ -43,11 +43,12 @@ def __init__(self, service_dir, configs):
             'cluster_name': CONF.stackanetes['cluster_name'],
             'external_ip': CONF.stackanetes['external_ip'],
             'interface_name': CONF.stackanetes['minion_interface_name'],
-            'namespace': CONF.stackanetes['namespace'],
+            'namespace': configs['namespace'],
             'monitors': CONF.ceph['ceph_mons'],
             'ceph_admin_keyring': CONF.ceph['ceph_admin_keyring'],
             'uuid': CONF.ceph['uuid'],
         }
+        LOG.error("configmap: {}".format(self.variables['namespace']))
```
Comment (mateusz-blaszkowski) [inline stackanetes/configmap.py]: This log should be removed

File: stackanetes/k8s_instance.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def __init__(self, service_name, service_dir):
         self.service_type = service_name.split("-")[0]
         self.service_dir = service_dir
         self._load_service_variables()
-        if not check_if_namespace_exist(CONF.stackanetes.namespace):
+        if not check_if_namespace_exist(self.namespace):
```
Comment (mateusz-blaszkowski) [inline stackanetes/k8s_instance.py]: `check_if_namespace_existS` (although I would prefer to name the method `namespace_exists`)

File: stackanetes/k8s_instance.py [hunk 3]
```diff
@@ -55,22 +55,26 @@ def _load_service_variables(self):
         self.configs = variables
         self.files = variables.get('files', [])
         self.containers = variables.get('containers', [])
+        self.namespace = variables.get('namespace', CONF.stackanetes.namespace)
+        LOG.error("k8s_instances: {}".format(self.namespace))
```
Comment (mateusz-blaszkowski) [inline stackanetes/k8s_instance.py]: This log should be removed

File: stackanetes/k8s_instance.py [hunk 4]
```diff
@@ -55,22 +55,26 @@ def _load_service_variables(self):
...
 
     def _load_configmaps(self):
         if self.containers:
             files = []
             [files.extend(x.get('files', [])) for x in self.containers]
-            for file in files:
+            for file_configuration in files:
```
Comment (mateusz-blaszkowski) [inline stackanetes/k8s_instance.py]: `configuration_file`

File: stackanetes/k8s_instance.py [hunk 5]
```diff
@@ -55,22 +55,26 @@ def _load_service_variables(self):
...
-                    file['configmap_name'], self.service_name))
-                configmap = ConfigMap(self.service_dir, file)
+                    file_configuration['configmap_name'], self.service_name))
+                file_configuration['namespace'] = self.namespace
+                configmap = ConfigMap(self.service_dir, file_configuration)
                 configmap.remove()
                 configmap.upload()
 
-        for file in self.files:
+        for file_configuration in self.files:
```
Comment (mateusz-blaszkowski) [inline stackanetes/k8s_instance.py]: the same as above

File: stackanetes/manifest.py [hunk 6]
```diff
@@ -42,7 +43,9 @@ def __init__(self, configuration, service_dir):
         self.external_ip = CONF.stackanetes.external_ip
         self.image_prefix = CONF.stackanetes.image_prefix
         self.host_network = configuration.get('host_network', False)
-        self.namespace = CONF.stackanetes.namespace
+        self.namespace = configuration.get('namespace',
+                                           CONF.stackanetes.namespace)
+        LOG.error("manifest: {}".format(self.namespace))
```
Comment (mateusz-blaszkowski) [inline stackanetes/manifest.py]: This log should be removed

**Comment by PiotrProkop** [comment]
LGTM.

**Pull request merged**